products/idn/api/patch-requests.md

```diff
@@ -0,0 +1,637 @@
+  },
+  {
+    "op": "add",
+    "path": "/connectorAttributes/filterString",
+    "value": "!( id.contains( \"m\" ) ) || !( id.contains( \"d\" ) )"
+  }
+]
+
+```
+
+This example request uses a "replace" to update the source's description and an "add" to add a filter string to the source's connector. 
+
+## Specify an operation 
+
+Once you know the ID of the resource you want to update and you have the resource's details, you can start writing your PATCH request. 
+
+The first step is to specify an operation to apply to the target resource. 
+
+Operation objects must have exactly one "op" member, whose value indicates the operation to perform. 
+
+These are the available PATCH operations: 
+
+- [Add](#add)
+- [Remove](#remove)
+- [Replace](#replace)
+- [Move](#move)
+- [Copy](#copy) 
+- [Test](#test)
+
+### Add 
+
+The "add" operation adds a value to the target location. For more information about the "add" operation and how it behaves in different scenarios, refer to the [JSON PATCH documentation](https://datatracker.ietf.org/doc/html/rfc6902). 
+
+This example uses the [Patch Source Schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to add a new "office" attribute to the end of a source schema's array of attributes:
+
+```json
+[
+  {
+    "op": "add", 
+    "path": "/attributes/-", 
+    "value": 
+    {
+      "name": "office", 
+      "type": "STRING", 
+      "schema": null, 
+      "description": "Office Location", 
+      "isMulti": false, 
+      "isEntitlement": false, 
+      "isGroup": false
+    }
+  }
+]
+```
+
+If you are adding a new value to an array of values, you can specify the position within the array where you want to add the new value. In this example, using the "-" expression at the end of the path specifies that the new attribute will be added to the end of the array of attributes. 
+
+You can use "0" to add a value to the beginning of the array. You can use "1" to add a value to the second positon, and so on. Using "-" adds the value to the end of the array. 
+
+### Remove 
+
+The "remove" operation removes a value from the target location. The target location must exist for the operation to be successful. 
+
+This example uses the [Patch Source](https://developer.sailpoint.com/idn/api/v3/update-source) endpoint to remove an existing filter string from a source's connector:
+
+```json
+ [
+  {
+    "op": "remove", 
+    "path": "/connectorAttributes/filterString", 
+    "value": "!( id.contains( \"m\" ) )"
+  }
+ ] 
+ ```
+
+ In this example, the PATCH request is removing a value that contains a filter string that excludes all connector IDs that have the letter "m" in them. 
+
+ The request will only remove a filter string matching the one specified, so any other filter strings or connector attributes will be unaffected. 
+ 
+ If there is no value matching the filter string specified by the PATCH request, the request will fail and you will receive an error. 
```

Comment:
It might be good to have two examples for remove.  Keep your first example, but also show an example of removing an item from a list.  For example:

```json
[{ "op": "remove", "path": "/accounts/0" }]
```